cirq-core/cirq/devices/fidelity.py

```diff
@@ -0,0 +1,216 @@
+from typing import Sequence
+from itertools import product
+import cirq
+import numpy as np
+from cirq.devices.noise_model import validate_all_measurements
+
+
+class Fidelity:
+    def __init__(
+        self,
+        *,
+        t1: float = None,
+        decay_constant: float = None,
+        xeb_fidelity: float = None,
+        pauli_error: float = None,
+        p00: float = None,
+        p11: float = None,
+    ) -> None:
+        """Creates a Fidelity object using the provided metrics
+
+        Args:
+          t1: t1 decay constant in ns
+          decay_constant: depolarization decay constant
+          xeb_fidelity: 2-qubit XEB Fidelity
+          pauli_error: total Pauli error
```

Comment:
It's not clear what system decay_constant and pauli_error refer to? Is this specifically only for 2-qubit gates?